site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -275,12 +276,24 @@ That means that you skipped the `--cleanup` flag and you should either delete th
 
 ## Controller TLS Certificates
 
+{{% feature expiryVersion="1.21.0" %}}
```

Comment:
Since we didn't merge this PR before the last release, this needs to be updated to the next release (1.22).